src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -857,6 +857,17 @@ public void switchFolder(FolderElement switchTo) {
 
 		upFolder.setEnabled(!currentFolder.isRoot());
 	}
+	
+	private void filter(String text) {
+		String filter = "f:" + text + " ";

```

Comment:
This implementation will not handle well filter at the end of text if it does not have additional space after.

Reply:
What do you mean?

Reply:
Manually write f:locked, for example, and try to remove it by clicking button. Make sure to not enter any whitespace

Reply:
ahhh yes
you are very smart xD